README.md

```diff
@@ -8,14 +8,11 @@ Project Intro
 </p>
 
 
-
 This is a project overview of a design guide for bitcoin with early thoughts on what it could be.
 
 Like a travel guide, it provides an introduction to a “foreign place”, shows you the highlights and tells you some of the best stories. It provides facts and maps to figure out how to get around, ideas for tours and things to do. Whether you end up taking the official museum tour, clubbing all night or go on shopping sprees is still your choice. Note that these are guides, not mandates. Just like in a foreign city, you can choose to go off the path, and forge a new direction. Others may just follow in the future.
 
-Comments on a previous version have been reviewed and used to improve this doc where relevant.
-
-A lot of discussion about it is taking place in the [bitcoin design Slack](https://bitcoindesign.slack.com/join/shared_invite/zt-fljgbpg0-wHr_thYUYWh5105UnGTWHw#/) (#bitcoin-design-guide).
+Discussion about it is taking place in the [bitcoin design Slack](https://bitcoindesign.slack.com/join/shared_invite/zt-fljgbpg0-wHr_thYUYWh5105UnGTWHw#/) (#bitcoin-design-guide).
```

Comment:
The invite link expired, it's no longer valid. The problem seems to be present in the original Google Doc so should not block the merging of this PR.